mitodl/mitxonline
# Course Product Page: show more enrollment dates

# What are the relevant tickets?
Fix https://github.com/mitodl/hq/issues/2393

# Description (What does it do?)
Shows more start dates for the course with a "Show Dates" toggle.

# Screenshots (if appropriate):
<img width="415" alt="Screen Shot 2023-09-26 at 9 28 50 PM" src="https://github.com/mitodl/mitxonline/assets/7574259/9b5d30c0-4d59-4c92-a996-afa8b5eff76f">


# How can this be tested?
Create two course runs for a course with enrollment open or in the future. Go to the course product page and toggle the "more dates" button to see more dates.
If there is only one option the "More Dates" button should not show.
